*.inspired.is-cool.dev.json

```diff
@@ -0,0 +1,17 @@
+{
+    "description": "Wildcard for Rammerhead",
+
+    "domain": "is-cool.dev",
+    "subdomain": "*.inspired",
```

Comment:
Wildcard domains (like *.example.is-not-a.dev) are supported too, but the reason for their registration should be very clear and described in detail.